src/helpers/updateContact.spec.js

```diff
@@ -0,0 +1,66 @@
+import {
+  updateContact,
+  getContactsToUpdate,
+  getLastSuccessOfService
+} from './updateContact'
+import { johnDoeContact } from './testData'
+
+import CozyClient from 'cozy-client'
+const client = new CozyClient({})
+
+describe('checkAndUpdate', () => {
```

Comment:
As Quentin said earlier, checkAndUpdate is not very telling. Are we normalizing the contact here ?

Reply:
Yes I think we can use this to normalize contacts. Then if we add other attributes in contacts, we juste have to update this service, don't you think ?

Reply:
Yes I agree.